pkg/defaultcomponents/defaults.go

```diff
@@ -78,7 +81,7 @@ func Components() (component.Factories, error) {
 		errs = multierr.Append(errs, err)
 	}
 
-	receivers, err := component.MakeReceiverFactoryMap(
+	receivers, err := receiver.MakeFactoryMap(
```

Comment:
this was done for receivers/exporters/extensions but not processors, were those changes to the processor package upstream just not yet included yet in 0.67?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.67.0 release notes do not indicate those deprecation's being made for processors yet.